analytics.go

```diff
@@ -63,3 +64,29 @@ func (c *Client) GetAggregatedIncidentData(ctx context.Context, analytics Analyt
 	err = c.decodeJSON(resp, &analyticsResponse)
 	return analyticsResponse, err
 }
+
+func (c *Client) GetAggregatedServiceData(ctx context.Context, analytics AnalyticsRequest) (AnalyticsResponse, error) {
+	var analyticsResponse AnalyticsResponse
+	headers := make(map[string]string)
+	headers["X-EARLY-ACCESS"] = "analytics-v2"
+
+	resp, err := c.post(ctx, "/analytics/metrics/incidents/services", analytics, headers)
+	if err != nil {
+		return analyticsResponse, err
+	}
+	err = c.decodeJSON(resp, &analyticsResponse)
+	return analyticsResponse, err
+}
+
+func (c *Client) GetAggregatedTeamData(ctx context.Context, analytics AnalyticsRequest) (AnalyticsResponse, error) {
+	var analyticsResponse AnalyticsResponse
+	headers := make(map[string]string)
+	headers["X-EARLY-ACCESS"] = "analytics-v2"
+
+	resp, err := c.post(ctx, "/analytics/metrics/incidents/teams", analytics, headers)
+	if err != nil {
+		return analyticsResponse, err
+	}
+	err = c.decodeJSON(resp, &analyticsResponse)
+	return analyticsResponse, err
+}
```

Comment:
Could you add a newline at the end of your files in this PR? I see it here and on some of the others too.

Reply:
Ah! Sorry about that. I gofmt'd the files. All modified files should have newline at the end now.